students/jesse_miller/session03/slicing_lab.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/python3
+#This function swaps the first and last characters in the tuple.  It works by
+#slicing the last character, adding position 1 through one back from last, and
+#then adding the first character.  All placed into the result variable.
+def exchange_first_last(seq):
+    result = seq[-1:] + seq[1:-1] + seq[0:1]
+    return result
```

Comment:
you can just do a `return seq[-1:] + seq[1:-1] + seq[0:1]` instead of assigning to a variable and then returning.